_posts/en/pages/2017-01-01-download.md

```diff
@@ -95,8 +95,7 @@ obtain_release_key: >
   key using this command:</p>
 
 choosing_builders: >
-  It is recommended that you choose a few individuals from this list who you find
-  trustworthy and import their keys as above, or import all the keys per the
+  It is recommended that you choose a few individuals from this list whose fingerprints you have confirmed to be authentic by comparing them on multiple trustworthy websites and import their keys as above, or import all the keys per the
   instructions in the <a href="$(BUILDER_KEYS_URL)"><code>contrib/builder-key</code>
   README</a>. You will later use their keys to check the signature attesting to the
```

Comment:
This file no longer exits. It might be better to directly link to the guix attestations repo and explain that each release is signed by a different set of builders. For example the 23.1 one is signed by https://github.com/bitcoin-core/guix.sigs/tree/main/23.1